stubs/jsonschema/jsonschema/validators.pyi

```diff
@@ -1,32 +1,18 @@
 from typing import Any
 
 from jsonschema import exceptions as exceptions
-from jsonschema.exceptions import ErrorTree as ErrorTree
-
-class _DontDoThat(Exception): ...
-
-validators: Any
-meta_schemas: Any
 
+def __getattr__(name): ...
```

Comment:
The `__getattr__` is used to provide deprecation warnings. As it causes every import from `jsonschema.validators` to be valid, we should leave it out of the stubs.